### ISSUE TITLE
实现问题

### ISSUE THREAD

**Issue by ZhenboZhao77** [zhengli97/CTKD#22]
你好作者！
再次打扰：
这是我的损失实现应该是普通的蒸馏：
def knowledge_distillation_kl_div_loss(pred,
                                       soft_label,
                                       T,
                                       class_reduction='mean',
                                       detach_target=True):
    r"""Loss function for knowledge distilling using KL divergence.

    Args:
        pred (Tensor): Predicted logits with shape (N, n + 1).
        soft_label (Tensor): Target logits with shape (N, N + 1).
        T (int): Temperature for distillation.
        detach_target (bool): Remove soft_label from automatic differentiation

    Returns:
        torch.Tensor: Loss tensor with shape (N,).
    """
    assert pred.size() == soft_label.size()
    target = F.softmax(soft_label / T, dim=1)
    if detach_target:
        target = target.detach()

    kd_loss = F.kl_div(
        F.log_softmax(pred / T, dim=1), target, reduction='none')
    if class_reduction == 'mean':
        kd_loss = kd_loss.mean(1)
    elif class_reduction == 'sum':
        kd_loss = kd_loss.sum(1)
    else:
        raise NotImplementedError
    kd_loss = kd_loss * (T * T)
    return kd_loss
请问是如何添加instance_temp?
因为我看到instance_temp是实现了一个Temp_MLP1，然后你说只需要更改distiller_zoo/KD.py的16-18行这个我不太能理解是怎样改的能否详细介绍一下？
谢谢作者

**Comment by ZhenboZhao77** [comment]
老师您好，请问我是应该直接使用models/temp_global.py输出的T把原来的T替换掉吗？
